XX-eltoo-transactions.md

```diff
@@ -114,22 +115,24 @@ where
    * `txin[0]` outpoint: `txid` and `output_index` from latest state `k` output (can be 0, the funding output)
    * `txin[0]` sequence: 0xFFFFFFFD, to allow [BIP125](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki#summary) replacement
    * `txin[0]` script bytes: 0
-   * `txin[0]` witness: `signature_for_inner_pubkey`
+   * `txin[0]` witness:
+     * control block: 0xc0 marker for tapscript, internel public key, merkle proof unless spending funding tx
+     * tapscript: EXPR_UPDATE_0 or EXPR_UPDATE(locktime), depending on what output is being spent
+     * `signature_for_inner_pubkey`
```

Comment:
I put this in the text, maybe insufficient:

```
We are ommitting all non-script related witness stack items for brevity such as control block, inner pubkeys, et al. A `<>` designates an empty vector as required for compliance with [BIP342](https://github.com/bitcoin/bips/blob/master/bip-0342.mediawiki) All taproot leaf versions are 0xC0 unless stated otherwise.
```

Reply:
It was mostly that you had an explicit control block entry at the same level as ins/outs which was confusing. Dropping the "control block" part entirely is fine, but having the tapscript you're using spelt out seems like it might be worthwhile.

Reply:
I'll take it.